CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2012, Codership Oy. All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+
+INCLUDE_DIRECTORIES(".")
+
+SET(WSREP_SOURCES wsrep_gtid.c wsrep_uuid.c wsrep_loader.c wsrep_dummy.c)
+
+ADD_CONVENIENCE_LIBRARY(wsrep ${WSREP_SOURCES})
+DTRACE_INSTRUMENT(wsrep)
```

Comment:
Also DTRACE_INSTRUMENT does not seem to be known by default. Should be wrapped with option WITH_DTRACE_INSTRUMENT or similar.

Reply:
`DTRACE_INSTRUMENT` is defined in MySQL's cmake/dtrace.cmake. I changed the code to:
```
IF(COMMAND DTRACE_INSTRUMENT)
  DTRACE_INSTRUMENT(wsrep)
ENDIF()
```
https://cmake.org/cmake/help/latest/command/if.html#command:if